steps/embrace-symbol-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,38 @@
+title: Embrace iOS Symbol Upload
+summary: |
+  Upload iOS debug symbols to Embrace through Bitrise CI integration
+description: |
+  Upload iOS debug symbols to Embrace through Bitrise CI. This will make your integration with Embrace more efficient. Environment Variables are supported via the upload tool.
+
+  Env Vars:
+  ---------
+  EMBRACE_APP: Your Embrace app idenifier
+  EMBRACE_API_TOKEN: Your Embrace upload API token
+website: https://github.com/embrace-io/bitrise-step-embrace-symbol-upload
+source_code_url: https://github.com/embrace-io/bitrise-step-embrace-symbol-upload
```

Comment:
This is 404 for me, it should be a public repo

Reply:
These should be updated properly now.